src/Libraries/Microsoft.Extensions.AI.Abstractions/Functions/AIFunctionArguments.cs

```diff
@@ -99,7 +106,7 @@ public object? this[string key]
     public void Clear() => _arguments.Clear();
 
     /// <inheritdoc />
-    bool ICollection<KeyValuePair<string, object?>>.Contains(KeyValuePair<string, object?> item) =>
+    public bool Contains(KeyValuePair<string, object?> item) =>
```

Comment:
Ditto

Reply:
Need to revert this and ignore the warning.